R/lint.R

```diff
@@ -68,9 +91,13 @@ lint <- function(filename, linters = NULL, cache = FALSE, ..., parse_settings =
 
   if (length(cache_path)) {
     lint_cache <- load_cache(filename, cache_path)
-    lints <- retrieve_file(lint_cache, filename, linters)
+    lint_obj <- if (is.null(text)) filename else list(
```

Comment:
nit: i would either put this all on one line (it looks like it fits in 120) or else but the `if` branch on its own line for readability (easier to read when both branches have similar indentation)

Reply:
Sure